promql/testdata/functions.test

```diff
@@ -385,6 +385,17 @@ eval instant at 1m stdvar_over_time(metric[1m])
 eval instant at 1m stddev_over_time(metric[1m])
   {} 3.249615
 
+# Tests for stddev_over_time and stdvar_over_time #4927.
```

Comment:
We should probably add a test for stddev too

Reply:
I'm confused, this test both stdvar_over_time and stddev_over_time

Reply:
We also have the stddev and stdvar aggregators over in engine.go